git-clone/entrypoint.sh

```diff
@@ -11,12 +11,29 @@ else
   LFS_SKIP_SMUDGE="--skip-smudge";
 fi
 
+# clear path
 rm -rf ${MOUNT_PATH}/*
 (rm -rf ${MOUNT_PATH}/.* || true)
+
+# set up git defaults
 git config --system push.default simple
-git lfs install $LFS_SKIP_SMUDGE --system
-git clone $REPOSITORY ${MOUNT_PATH}
+
+# extract the GitLab host and path
+pat='^(http[s]?:\/\/)([^:\/\s]+)\/?([a-zA-Z0-9_]+)?$'
```

Comment:
```suggestion
pat='^(http[s]?:\/\/)([^\/]+)\/?([a-zA-Z0-9_\/\-]+)?$'
```
This would also correctly match the following:
`https://some.host:port/git-lab/path`

Reply:
do we want this to be more generic?

Reply:
I would say so. Forbidding hyphens or slashes in the path seems like an unnecessary restriction to me, and allowing different ports might be helpful in some dev settings.

Reply:
yeah but then do you want a separate capture group for the port?